packages/common-all/src/util/treeUtil.ts

```diff
@@ -89,25 +87,20 @@ export class TreeUtils {
 
       notesLabelById[note.id] = title;
 
-      const fm = PublishUtils.getPublishFM(note);
-
       const treeMenuNode: TreeMenuNode = {
         key: note.id,
         title,
         icon,
         hasTitleNumberOutlined: note.fname.startsWith(TAGS_HIERARCHY),
         vaultName: VaultUtils.getName(note.vault),
-        navExclude: fm.nav_exclude || false,
         children: [],
       };
 
       if (child2parent[note.id] === undefined) {
         child2parent[note.id] = parent;
       }
 
-      const shouldIgnoreChildren = fm.nav_exclude_children || fm.has_collection;
-
-      if (!shouldIgnoreChildren && sidebarItem.type === "category") {
```

Comment:
Moved into `sidebar.ts` so that logic that dictates the the sidebar output is placed in a single location.